### DISCUSSION TITLE
passwords-addon needs php update 8.1

### DISCUSSION THREAD

**Discussion by janprive** [nextcloud/all-in-one#1758] [Questions] [answered]
Is this php updated automatically by docker? Must I update php by hand?

**Comment by Zoey2936** [comment, marked as answer]
PHP is currently set to v8.0, this cant be changed by you
The last state I read was that AIO upgrades to PHP v8.1 when NC v26 is released (https://github.com/nextcloud/all-in-one/discussions/1396#discussioncomment-4262553)

**Reply by janprive** [reply]
Thanks!!
